### DISCUSSION TITLE
Option to change the width of the sidebar

### DISCUSSION THREAD

**Discussion by SebastiaanKloos** [filamentphp/filament#1252] [Ideas]
In some of my projects, I have a lot of columns with large forms. Being able to change the size of the sidebar in a config file would be incredible awesome.

<img width="345" alt="CleanShot 2022-01-16 at 12 02 56@2x" src="https://user-images.githubusercontent.com/6506510/149657265-5376cd5d-46be-4de0-b0ae-bdff5f38c4d2.png">

**Comment by danharrin** [comment]
Sure, I'd merge a config option for this. Maybe we could use `md`, `lg`, `xl` (`md` as default) in the config.

**Reply by SebastiaanKloos** [reply]
Yeah but also be able to downscale the sidebar. As you see in my example, the sidebar is much wider than the content

**Reply by danharrin** [reply]
Sure - an `sm` option too.
